### ISSUE TITLE
redirect all rewards for all badger multisigs to treasury ops wallet

### ISSUE THREAD

**Issue by mitche50** [Badger-Finance/badger-rewards#539]
this is in accordance with dapp. instead of trying to time the claim call perfectly on the dev msig (needs to be posted and execd within one cycle), dapp proposed to redirect rewards to an ops wallet instead.

treasury ops (`0x042B32Ac6b453485e357938bdC38e0340d4b9276`) will be able to post and execute much faster, plus it is also the msig that will process those rewards if needed (dogfood, lock, etc).

relevant msigs, taken from https://github.com/Badger-Finance/badger-multisig/blob/main/helpers/addresses.py :
```
        "dev_multisig": "0xB65cef03b9B89f99517643226d76e286ee999e77",
        "test_multisig": "0x33909cb2633d4B298a72042Da5686B45E9385ed0",
        "techops_multisig": "0x86cbD0ce0c087b482782c181dA8d191De18C8275",
        "politician_multisig": "0x6F76C6A1059093E21D8B1C13C4e20D8335e2909F",
        "recovered_multisig": "0x9faA327AAF1b564B569Cb0Bc0FDAA87052e8d92c",
        "ops_multisig": "0xD4868d98849a58F743787c77738D808376210292",
        "ops_multisig_old": "0x576cD258835C529B54722F84Bb7d4170aA932C64",
        "treasury_vault_multisig": "0xD0A7A8B98957b9CD3cFB9c0425AbE44551158e9e",
```

note that the ibbtc multisig should explicitly NOT be included in this and should keep claiming its own rewards.

also i just saw that the tree has claimable assets itself..? https://api.badger.finance/v2/reward/tree/0x660802Fc641b154aBA66a62137e71f331B6d787A

finally, maybe this is a good chance to also make sure all msigs are blacklisted from native rewards (not sure if that can be fixed retroactively?)

**Comment by mitche50** [comment]
Closed by #668

**Comment by jorijnsmit** [comment]
voter msig needs to be added to this; `0xA9ed98B5Fb8428d68664f3C5027c62A10d45826b`.

not sure if you want to create a new ticket for this @mitche50 @Darruma, can i leave that up to you..?
